tests/Model/ModelManagerTest.php

```diff
@@ -776,6 +776,42 @@ public function testModelReverseTransform(): void
         $this->assertSame('hello', $object->getMultiWordProperty());
     }
 
+    /**
+     * NEXT_MAJOR: Remove this test.
```

Comment:
why should we remove this test fully? We should probably remove `testModelReverseTransform` then?

Reply:
Yes indeed, wrong copy paste